src/agent/onefuzz-agent/src/tasks/heartbeat.rs

```diff
@@ -146,15 +81,15 @@ pub trait HeartbeatSender {
     }
 }
 
-impl HeartbeatSender for HeartbeatClient {
+impl HeartbeatSender for HeartbeatClient<HeartbeatData> {
     fn send(&self, data: HeartbeatData) -> Result<()> {
         let mut messages_lock = self.messages.lock().unwrap();
```

Comment:
This isn't new, but is there a reason these are calling `unwrap()`, when they return a `Result`?